### DISCUSSION TITLE
萌娘百科 搜索栏重影问题

### DISCUSSION THREAD

**Discussion by Mineocean** [F9y4ng/GreasyFork-Scripts#359] [问答专区(Question / Answer)] [answered]
浏览器：Firefox 128.0
脚本管理器：Tampermonkey v5.1.1
脚本版本：字体渲染（自用脚本）2024.07.06.1
问题网站：https://zh.moegirl.org.cn/Mainpage#/flow
![图片](https://github.com/user-attachments/assets/2cff3c71-e29b-4910-83e9-8f6b62db7037)
![图片](https://github.com/user-attachments/assets/dae93cb2-85bf-4e6e-9335-ecb5bd3aa4ff)

**Comment by F9y4ng** [comment, marked as answer]
在**排除渲染的HTML标签**中，追加`,.n-input__input-el`，保存为站点独享数据。
